### PR TITLE
Questions and Answers widget

### PR DESCRIPTION
**Pull request by Katerina-spb**
Created file structure and hard coded the components.
Just keeping you guys updated on where I am😄

Screen Shot 2021-03-05 at 4 07 46 PM

### CODE REVIEW

File: src/components/questions/search/Search.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+var Search = () => {
+  return (
+    <from>
```
Comment (Hkanabar) [inline src/components/questions/search/Search.js]: Is this a typo? Should be form not from.

**Review by Hkanabar — COMMENTED** [review]
Looks mostly good. You've got a solid baseline for what you want to do. There was just one minor thing which seems to be a typo, but other than that nice work!

File: src/components/questions/search/Search.js (hunk 1 above)
Comment (Katerina-spb) [reply src/components/questions/search/Search.js]: 🤣 thanks!

**Comment by chaines** [comment]
Looks good to me!

**Pull request merged**